### PR TITLE
Tweak GDScript error messages for consistency

### PR DESCRIPTION
**Pull request by Calinou**
Follow-up to #31607.

This also mentions the number of arguments given when too many (or not enough) arguments were passed to a function.

### CODE REVIEW

File: modules/gdscript/gdscript_parser.cpp [hunk 1]
```diff
@@ -515,13 +515,13 @@ GDScriptParser::Node *GDScriptParser::_parse_expression(Node *p_parent, bool p_s
...
 			if (tokenizer->get_token() != GDScriptTokenizer::TK_PARENTHESIS_CLOSE) {
-				_set_error("Expected ')' after 'preload' path");
+				_set_error("Expected \")\" after \"preload\" path.");
 				return NULL;
 			}
 
 			Ref<GDScript> gds = res;
 			if (gds.is_valid() && !gds->is_valid()) {
-				_set_error("Couldn't fully preload the script, possible cyclic reference or compilation error. Use \"load()\" instead if a cyclic reference is intended.");
+				_set_error("Couldn\"t fully preload the script, possible cyclic reference or compilation error. Use \"load()\" instead if a cyclic reference is intended.");
```
Comment (YeldhamDev) [inline modules/gdscript/gdscript_parser.cpp]: "Couldn " t"?

Comment (Calinou) [reply modules/gdscript/gdscript_parser.cpp]: Sorry, I used an automatic replacement tool and missed some errors it introduced :slightly_smiling_face:

I fixed them.

**Comment by akien-mga** [comment]
Needs a rebase.

We probably need to have a discussion and documented consensus on the quote style for errors, I see you've changed a lot of single quotes to double quotes, but this hasn't really been discussed.

It does make the code less readable as we need to escape them, but if there's consensus that the output is better, that's fine by me.

**Comment by Calinou** [comment]
> It does make the code less readable as we need to escape them, but if there's consensus that the output is better, that's fine by me.

Maybe we could use C++11's raw string literals if there are more than two characters to escape:

```cpp
R"(Some string that accepts "double quotes" or even line breaks.)"
```

**Comment by vnen** [comment]
> We probably need to have a discussion and documented consensus on the quote style for errors, I see you've changed a lot of single quotes to double quotes, but this hasn't really been discussed.
> 
> It does make the code less readable as we need to escape them, but if there's consensus that the output is better, that's fine by me.

IMO it's more natural to see `"` and it's easier to make out what's surrounded by the quotes. It's not widely used probably just because it's hard to escape.

**Comment by vnen** [comment]
This was made before the rewrite, so either should be closed or change the target to 3.x (assuming we want to change the style there). If there's something still on master that's inconsistent it's better to open a new PR.

**Comment by akien-mga** [comment]
Closing as per above comment.

**Pull request closed without merging**